sympy/physics/control/lti.py

```diff
@@ -735,11 +775,47 @@ class Series(Basic):
     >>> S4.doit()
     TransferFunction((s**3 - 2)*(-p**2*(-p + s) + (p + s)*(a*p**2 + b*s)), (-p + s)*(p + s)*(s**4 + 5*s + 6), s)
 
+    SISO-System Examples -
+
+    >>> mat_a = Matrix([[5*s], [5]])  # 2 Outputs 1 Input
+    >>> mat_b = Matrix([[5, 1/(6*s**2)]])  # 1 Output 2 Inputs
+    >>> mat_c = Matrix([[1, s], [5/s, 1]])  # 2 Outputs 2 Inputs
+    >>> tfm_a = TransferFunctionMatrix.from_Matrix(mat_a, s)
+    >>> tfm_b = TransferFunctionMatrix.from_Matrix(mat_b, s)
+    >>> tfm_c = TransferFunctionMatrix.from_Matrix(mat_c, s)
+    >>> Series(tfm_c, tfm_b, tfm_a)
+    Series(TransferFunctionMatrix(((TransferFunction(1, 1, s), TransferFunction(s, 1, s)), (TransferFunction(5, s, s), TransferFunction(1, 1, s)))), TransferFunctionMatrix(((TransferFunction(5, 1, s), TransferFunction(1, 6*s**2, s)),)), TransferFunctionMatrix(((TransferFunction(5*s, 1, s),), (TransferFunction(5, 1, s),))))
+    >>> pprint(_, use_unicode=False)  #  For Better Visualization
```

Comment:
Let's pass `use_unicode=True` in the docstrings? It's much readable for the users.
```python
In [76]: G1 = TransferFunctionMatrix.from_Matrix(Matrix([[5*s], [5]]), s)                 

In [77]: G2 = TransferFunctionMatrix.from_Matrix(Matrix([[5, 1/(6*s**2)]]), s)            

In [78]: G3 = TransferFunctionMatrix.from_Matrix(Matrix([[1, s], [5/s, 1]]), s)           

In [79]: pprint(Series(G3, G2, G1))                                                       
⎡5⋅s⎤             ⎡1  s⎤ 
⎢───⎥             ⎢─  ─⎥ 
⎢ 1 ⎥             ⎢1  1⎥ 
⎢   ⎥  ⎡5   1  ⎤  ⎢    ⎥ 
⎢ 5 ⎥  ⎢─  ────⎥  ⎢5  1⎥ 
⎢ ─ ⎥  ⎢1     2⎥  ⎢─  ─⎥ 
⎣ 1 ⎦τ⋅⎣   6⋅s ⎦τ⋅⎣s  1⎦τ
```
Dot should be placed slightly above; notice how it is done here:
```python
In [27]: mat_a = Matrix([[5*s], [5]])                                                 

In [28]: mat_b = Matrix([[5, 1/(6*s**2)]])                                                                              
                                                  
In [32]: pprint(MatMul(mat_a, mat_b))                                                                    
      ⎡    1  ⎤
⎡5⋅s⎤ ⎢5  ────⎥
⎢   ⎥⋅⎢      2⎥
⎣ 5 ⎦ ⎣   6⋅s ⎦
```

Reply:
> Let's pass `use_unicode=True` in the docstrings? It's much readable for the users.

It cannot be done, in my opinion. I've already tried asking for a solution in the past -

https://github.com/sympy/sympy/pull/21556#issuecomment-862621542

I also tried to lookup for Unicode-containing `pprint` examples in docstrings, but I couldn't find one.

Reply:
> Dot should be placed slightly above; notice how it is done here:

I had already noticed that and tried to make it equivalent to the `MatMul`'s output but even after multiple attempts, couldn't get it done.

Reply:
> I also tried to lookup for Unicode-containing pprint examples in docstrings, but I couldn't find one.

```
$ git grep ">>> pprint(" | grep "use_unicode=True)"
doc/src/modules/rewriting.rst:    >>> pprint(cse(sqrt(sin(x))), use_unicode=True)
doc/src/modules/rewriting.rst:    >>> pprint(cse(sqrt(sin(x)+5)*sqrt(sin(x)+4)), use_unicode=True)
doc/src/modules/rewriting.rst:    >>> pprint(cse((x-y)*(z-y) + sqrt((x-y)*(z-y))), use_unicode=True)
doc/src/modules/solvers/solveset.rst:   >>> pprint(ImageSet(Lambda(n, 2*pi*n), S.Integers), use_unicode=True)
doc/src/modules/solvers/solveset.rst:   >>> pprint(ComplexRegion(FiniteSet(1)*Interval(0, 2*pi), polar=True), use_unicode=True)
doc/src/modules/solvers/solveset.rst:   >>> pprint(ComplexRegion(Interval(-oo, oo)*Interval(0, oo)), use_unicode=True)
doc/src/modules/solvers/solveset.rst:    >>> pprint(solveset(abs(x) - n, x, domain=S.Reals), use_unicode=True)
doc/src/tutorial/printing.rst:    >>> pprint(Integral(sqrt(1/x), x), use_unicode=True)
sympy/physics/optics/polarization.py:>>> pprint(x0, use_unicode=True)
sympy/physics/optics/polarization.py:>>> pprint(s0, use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(jones_vector(psi, chi), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(jones_vector(0, 0), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(jones_vector(pi/2, 0), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(jones_vector(pi/4, 0), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(jones_vector(-pi/4, 0), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(jones_vector(0, pi/4), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(jones_vector(0, -pi/4), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(stokes_vector(psi, chi, p, I), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(stokes_vector(0, 0), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(stokes_vector(pi/2, 0), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(stokes_vector(pi/4, 0), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(stokes_vector(-pi/4, 0), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(stokes_vector(0, pi/4), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(stokes_vector(0, -pi/4), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(stokes_vector(0, 0, 0), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(J, use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(R, use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(HWP, use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(QWP, use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(NDF, use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(reflective_filter(R), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(mueller_matrix(linear_polarizer(theta)), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(mueller_matrix(half_wave_retarder(theta)), use_unicode=True)
sympy/physics/optics/polarization.py:    >>> pprint(mueller_matrix(quarter_wave_retarder(theta)), use_unicode=True)
sympy/physics/wigner.py:    >>> pprint(wigner_d_small(half, beta), use_unicode=True)
sympy/physics/wigner.py:    >>> pprint(wigner_d_small(2*half, beta), use_unicode=True)
sympy/physics/wigner.py:    >>> pprint(wigner_d_small(half, beta).subs({beta:pi/2}), use_unicode=True)
sympy/physics/wigner.py:    >>> pprint(wigner_d(half, alpha, beta, gamma), use_unicode=True)
```

Reply:
There needs to be a decision about using unicode in docstrings. It is deliberately not used in most of the codebase even if there are some exceptions.

Reply:
The unicode test seems to be entirely based around conventions from python 2:

https://github.com/sympy/sympy/blob/cc67e93c1e05a46fb3d5c4f445086aa89dc09353/sympy/testing/quality_unicode.py#L5-L28

Reply:
I encourage not to use unicode in source files and docstrings because they need special monospace fonts to look not misaligned.